plugins/cpp_metrics/model/include/model/cppastnodemetrics.h

```diff
@@ -17,9 +17,10 @@ struct CppAstNodeMetrics
   {
     PARAMETER_COUNT = 1,
     MCCABE = 2,
-    BUMPY_ROAD = 3,
-    LACK_OF_COHESION = 4,
-    LACK_OF_COHESION_HS = 5,
+    MCCABE_TYPE = 3,
```

Comment:
In this case I suggest the `MCCABE` to be renamed as well.

Reply:
Solved in [62406a3](https://github.com/Ericsson/CodeCompass/pull/727/commits/62406a35283dc77fc5046950a96f49f252c9dbda).